### ISSUE TITLE
REST API is not working

### ISSUE THREAD

**Issue by SniperCZE** [containous/traefik#2944]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

I would like to use REST API to on-the-fly reconfiguration.

### What did you expect to see?

REST API is accessible and can configure Traefik with it

### What did you see instead?

Every API call ends with error 404

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
1.5.3
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
loglevel = "DEBUG"

[api]
dashboard = true

[rest]
entryPoint = "traefik"
```
API call from clients:
```
curl -XPUT "http://lb-tester.tryadmins.net:8080/api" -d '{ "frontends" : { "frontend1" : {"backend" : "backend1" } }, "backends" : { "backend1" : { "servers" : { "server1" : { "URL" : "www.google.com" } } } } }'
404 page not found
```

### If applicable, please paste the log output in debug mode (`--debug` switch)

```
INFO[2018-02-27T16:46:30+01:00] Using TOML configuration file /root/traefik_rest.toml 
INFO[2018-02-27T16:46:30+01:00] Traefik version v1.5.2 built on 2018-02-12_10:56:31AM 
INFO[2018-02-27T16:46:30+01:00] 
Stats collection is disabled.
Help us improve Traefik by turning this feature on :)
More details on: https://docs.traefik.io/basics/#collected-data
 
DEBU[2018-02-27T16:46:30+01:00] Global configuration loaded {"LifeCycle":{"RequestAcceptGraceTimeout":0,"GraceTimeOut":0},"GraceTimeOut":0,"Debug":false,"CheckNewVersion":true,"SendAnonymousUsage":false,"AccessLogsFile":"","AccessLog":null,"TraefikLogsFile":"","TraefikLog":null,"LogLevel":"DEBUG","EntryPoints":{"http":{"Network":"","Address":":80","TLS":null,"Redirect":null,"Auth":null,"WhitelistSourceRange":null,"Compress":false,"ProxyProtocol":null,"ForwardedHeaders":{"Insecure":true,"TrustedIPs":null}},"traefik":{"Network":"","Address":":8080","TLS":null,"Redirect":null,"Auth":null,"WhitelistSourceRange":null,"Compress":false,"ProxyProtocol":null,"ForwardedHeaders":{"Insecure":true,"TrustedIPs":null}}},"Cluster":null,"Constraints":[],"ACME":null,"DefaultEntryPoints":["http"],"ProvidersThrottleDuration":2000000000,"MaxIdleConnsPerHost":200,"IdleTimeout":0,"InsecureSkipVerify":false,"RootCAs":null,"Retry":null,"HealthCheck":{"Interval":30000000000},"RespondingTimeouts":null,"ForwardingTimeouts":null,"Web":null,"Docker":null,"File":null,"Marathon":null,"Consul":null,"ConsulCatalog":null,"Etcd":null,"Zookeeper":null,"Boltdb":null,"Kubernetes":null,"Mesos":null,"Eureka":null,"ECS":null,"Rancher":null,"DynamoDB":null,"ServiceFabric":null,"Rest":{"EntryPoint":"traefik","CurrentConfigurations":null},"API":{"EntryPoint":"traefik","Dashboard":true,"Debug":false,"CurrentConfigurations":null,"Statistics":null},"Metrics":null,"Ping":null} 
INFO[2018-02-27T16:46:30+01:00] Preparing server http &{Network: Address::80 TLS:<nil> Redirect:<nil> Auth:<nil> WhitelistSourceRange:[] Compress:false ProxyProtocol:<nil> ForwardedHeaders:0xc4202c2080} with readTimeout=0s writeTimeout=0s idleTimeout=3m0s 
INFO[2018-02-27T16:46:30+01:00] Preparing server traefik &{Network: Address::8080 TLS:<nil> Redirect:<nil> Auth:<nil> WhitelistSourceRange:[] Compress:false ProxyProtocol:<nil> ForwardedHeaders:0xc4202c20c0} with readTimeout=0s writeTimeout=0s idleTimeout=3m0s 
INFO[2018-02-27T16:46:30+01:00] Starting server on :80                       
INFO[2018-02-27T16:46:30+01:00] Starting provider *rest.Provider {"EntryPoint":"traefik","CurrentConfigurations":{}} 
INFO[2018-02-27T16:46:30+01:00] Starting server on :8080                     
^CINFO[2018-02-27T16:50:46+01:00] I have to go... interrupt                    
INFO[2018-02-27T16:50:46+01:00] Stopping server gracefully                   
DEBU[2018-02-27T16:50:46+01:00] Waiting 0s seconds before killing connections on entrypoint traefik... 
DEBU[2018-02-27T16:50:46+01:00] Waiting 0s seconds before killing connections on entrypoint http... 
DEBU[2018-02-27T16:50:46+01:00] Entrypoint traefik closed                    
DEBU[2018-02-27T16:50:46+01:00] Entrypoint http closed                       
INFO[2018-02-27T16:50:46+01:00] Server stopped                               
INFO[2018-02-27T16:50:46+01:00] Shutting down
```

**Comment by traefiker** [comment]
Thanks for your interest in Traefik !

  Every question asked in issues means less features 😉.

  The issue tracker is for reporting bugs and feature requests only.
  For end-user related support questions, refer to one of the following:

  - [Stack Overflow](https://stackoverflow.com/questions/tagged/traefik) using the "traefik" tag
  - the [Traefik community Slack channel](https://traefik.herokuapp.com)

**Comment by SniperCZE** [comment]
This is not support question. As you can see in bugreport, toml configuration is corresponding with documentation at https://docs.traefik.io/configuration/backends/rest/ so I think I can expect working REST API with this settings, which is not true (see curl request with 404 error code). This leads me to think there is a bug in traefik or in documentation and this should be fixed.

**Comment by mmatur** [comment]
@SniperCZE You need to change your curl request like that

```console
curl -XPUT "http://lb-tester.tryadmins.net:8080/api/providers/rest" -d '{ "frontends" : { "frontend1" : {"backend" : "backend1" } }, "backends" : { "backend1" : { "servers" : { "server1" : { "URL" : "www.google.com" } } } } }'
```

**Comment by traefiker** [comment]
Closed by #2950.
